packages/styled-base/src/index.js

```diff
@@ -105,7 +105,17 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
                 classInterpolations,
                 props.className
               )
+            } else if (
```

Comment:
Maybe that wasn't clear - but in my last comment I've also included alternative solution to the "problem", the condition of this else if should be just `props.className != null` and the body of this shouldn't call `getRegisteredStyles`, like this:
```js
if (typeof props.className === 'string') {
  // ...
} else if (props.className != null) {
  className += `${props.className} `
}
```

Reply:
Ahhh okay, I see what you mean